src/components/dataTable/cell.scss

```diff
@@ -0,0 +1,65 @@
+$table-head-font-size: 12px;
+$table-body-font-size: 13px;
+
+table.md-table th,
+table.md-table td {
+  text-align: left;
+  white-space: nowrap;
+  vertical-align: middle;
+
+  &:first-child {
+    padding: 0 24px;
+  }
+
+  &:nth-child(n+2):nth-last-child(n+2) {
+    padding: 0 56px 0 0;
+  }
+
+  &:last-child {
+    padding: 0 24px 0 0;
+  }
+
+  &.md-align-left {
```

Comment:
I assume this is used by users..
Please add `md-align-start` and `md-align-end` to support rtl as well :)